gard_owl_ingest/analysis/mondo_mapping_status.py

```diff
@@ -21,9 +22,44 @@
 DATASOURCE_CSV = DATA_DIR / 'GARD_disease_list.csv'
 MONDO_SSSOM_TSV = TMP_INPUT_DIR / 'mondo.sssom.tsv'
 GARD_MONDO_SSSOM_TSV = TMP_INPUT_DIR / 'mondo_hasdbxref_gard.sssom.tsv'
+MAPPING_PREDICATES: List[MAPPING_PREDICATE] = [
+    'skos:exactMatch', 'skos:narrowMatch', 'skos:broadMatch', 'skos:relatedMatch']
 
-
+# todo: move this into gard_owl_ingest.py
 # noinspection PyUnresolvedReferences Note_becausePycharmDoesntKnowAboutNamedTuples
+def gard_native_mappings(df: pd.DataFrame = None) -> Dict[CURIE, Dict[MAPPING_PREDICATE, List[CURIE]]]:
```

Comment:
FYI: Created function `gard_native_mappings()` which is now used by analysis script and ingest script.